wpimath/src/main/native/include/frc/kinematics/SwerveDriveKinematics.h

```diff
@@ -146,7 +147,10 @@ class SwerveDriveKinematics {
    * @return The resulting chassis speed.
    */
   template <typename... ModuleStates>
-  ChassisSpeeds ToChassisSpeeds(ModuleStates&&... wheelStates) const;
+  requires(std::is_same_v<std::remove_reference_t<ModuleStates>,
+                          SwerveModuleState>&&...) ChassisSpeeds
```

Comment:
Not a big fan of this formatting, but if wpiformat agrees with it, I suppose it's fine.

Reply:
Even clang-format 15 seems to think requires() is a function and formats it like that.